src/StartProject.Asp/Startup.cs

```diff
@@ -8,14 +8,24 @@
 using Moryx.TestModule;
 using System.Text.Json.Serialization;
 using Microsoft.AspNetCore.Mvc.Controllers;
+using Microsoft.Extensions.Configuration;
 
 namespace StartProject.Asp
 {
     public class Startup
     {
+        public IConfiguration Config { get; }
```

Comment:
Do we need this in all applications from now on? 🤔

Reply:
If you also want to use ASP options: yes, otherwise no